docs/tutorials/analysis/1D/spectral_analysis.ipynb

```diff
@@ -549,11 +549,10 @@
    "outputs": [],
    "source": [
     "plt.figure(figsize=(8, 5))\n",
-    "flux_points.table[\"is_ul\"] = flux_points.table[\"ts\"] < 4\n",
-    "ax = flux_points.plot(\n",
-    "    energy_power=2, flux_unit=\"erg-1 cm-2 s-1\", color=\"darkorange\"\n",
-    ")\n",
-    "flux_points.to_sed_type(\"e2dnde\").plot_ts_profiles(ax=ax);"
+    "flux_points.table[\"is_ul\"] = flux_points.ts < 4\n",
```

Comment:
Shouldn't we get rid of this approach? A minima we have `sqrt_ts` which contains the sign, it is a better choice to use it once we have negative amplitudes.
If we define a level of significance for UL I suppose we might use it as the threshold to consider the point as an upper limit by default.

Reply:
I fully agree, the question is just where to keep the information on the sigma for ULs and errors. In a sense it maybe directly belongs to the `FluxEstimate` object as meta data. For `FluxPoints` it could be stored in the `table.meta` but for `FluxMaps` it is less clear...